jumpscale/packages/threebot_deployer/scripts/backup.sh

```diff
@@ -7,8 +7,8 @@ gzip -d rest-server-0.9.7-linux-amd64.gz
 chmod +x rest-server-0.9.7-linux-amd64
 mv /root/rest-server-0.9.7-linux-amd64 /usr/bin/rest-server
 
-apt-get install  -y apache2
+apt-get install -y apache2
```

Comment:
why do we need apache ?

Reply:
not sure how the backup servers work

Reply:
@waleedhammam that's for the htpasswd tool